cfretl/remote_settings.py

```diff
@@ -58,8 +59,9 @@ class CFRRemoteSettings:
     """
 
     def __init__(self):
-        self._kinto_uri = settings.KINTO_URI
-        self._kinto_bucket = settings.KINTO_BUCKET
+        self._kinto_bucket_path = "{base_uri:s}/buckets/main".format(
```

Comment:
```suggestion
        self._kinto_bucket_path = "{base_uri:s}/buckets/{bucket:s}".format(
```

Reply:
It's only used to boot strap the `cfr-ml-control` bucket during testing.